docs/dv/08_distributed-validator-cluster-manifest.md

```diff
@@ -21,41 +22,46 @@ The `charon create dkg` command is used to create `cluster-definition.json` file
 The `charon create cluster` command combines both steps into one and just outputs the final `cluster-lock.json` without a DKG step.
 
 The schema of the `cluster-definition.json` is defined as:
+
 ```json
 {
-  "name": "best cluster",                       // Optional cosmetic identifier
+  "name": "best cluster", // Optional cosmetic identifier
   "operators": [
     {
-      "address": "0x123..abfc",                 // ETH1 address of the operator
-      "enr": "enr://abcdef...12345",            // Charon node ENR
-      "nonce": 1,                               // Nonce (incremented each time the ENR is added/signed)
-      "config_signature": "123456...abcdef",    // EIP712 Signature of config_hash by ETH1 address priv key
-      "enr_signature": "123654...abcedf"        // EIP712 Signature of ENR by ETH1 address priv key
+      "address": "0x123..abfc", // ETH1 address of the operator
+      "enr": "enr://abcdef...12345", // Charon node ENR
+      "nonce": 1, // Nonce (incremented each time the ENR is added/signed)
```

Comment:
if this release contains def v1.2.0, we should probably remove the nonce i think, need to confirm

Reply:
+1